src/main/java/mate/academy/spring/mapper/impl/response/UserResponseMapper.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.spring.mapper.impl.response;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserResponseMapper implements DtoResponseMapper<UserResponseDto, User> {
+
+    @Override
+    public UserResponseDto toDto(User object) {
```

Comment:
Consider renaming the parameter 'object' to a more descriptive name, such as 'user'. This will improve code readability. 📘